### PR TITLE
jucs added

### PR DESCRIPTION
**Pull request by yegor256**
per #1528

### CODE REVIEW

**Comment by yegor256** [comment]
@volodya-lombrozo please, take a look

File: pom.xml [hunk 1]
```diff
@@ -197,6 +197,12 @@ SOFTWARE.
         <version>3.0.1u2</version>
         <scope>provided</scope>
       </dependency>
+      <dependency>
+        <groupId>org.eolang</groupId>
+        <artifactId>jucs</artifactId>
+        <version>0.0.2</version>
```
Comment (volodya-lombrozo) [inline pom.xml:203]: @yegor256 Maybe the first version have to be at least `0.1.2`? I'm not sure here but in the semver the third number is just a [bugfix](https://semver.org). I read that version as "bug fix for 0 minor version" - sounds strange.

File: eo-parser/src/test/java/org/eolang/parser/PacksTest.java [hunk 2]
```diff
@@ -41,15 +36,9 @@
 final class PacksTest {
 
     @ParameterizedTest
-    @MethodSource("yamlPacks")
+    @ClasspathSource(value = "org/eolang/parser/packs/", glob = "**/*.yaml")
     void testPacks(final String pack) throws Exception {
```
Comment (volodya-lombrozo) [inline eo-parser/src/test/java/org/eolang/parser/PacksTest.java:40]: @yegor256  `testPacks` -> `parsesPacks`

File: eo-maven-plugin/src/test/java/org/eolang/maven/SodgMojoTest.java [hunk 3]
```diff
@@ -74,16 +71,10 @@ void bigSlowTest() throws Exception {
     }
 
     @ParameterizedTest
-    @MethodSource("yamlPacks")
+    @ClasspathSource(value = "org/eolang/maven/sodgs/", glob = "*.yaml")
     @SuppressWarnings("PMD.JUnitTestContainsTooManyAsserts")
     void testPacks(final String pack) throws Exception {
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/test/java/org/eolang/maven/SodgMojoTest.java:76]: @yegor256  `testPacks` -> `generatesSodgForPacks`

File: eo-maven-plugin/src/test/java/org/eolang/maven/SnippetTest.java [hunk 4]
```diff
@@ -83,17 +80,11 @@ final class SnippetTest {
 
     @Disabled
     @ParameterizedTest
-    @MethodSource("yamlSnippets")
     @SuppressWarnings("unchecked")
+    @ClasspathSource(value = "org/eolang/maven/snippets/", glob = "*.yaml")
     void testFullRun(final String yml) throws Exception {
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/test/java/org/eolang/maven/SnippetTest.java:85]: `testFullRun` -> `runsAllSnippets`

File: eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java [hunk 5]
```diff
@@ -53,6 +56,15 @@
 @SuppressWarnings({"PMD.AvoidDuplicateLiterals", "PMD.TooManyMethods"})
 final class OptimizeMojoTest {
 
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/maven/packs/", glob = "**/*.yaml")
+    void testPacks(final String pack) throws Exception {
```
Comment (volodya-lombrozo) [inline eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java:61]: @yegor256 `testPacks` -> `checksOptimizstionsForPacks`

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@yegor256 I'm not sure about exact test names, but let's use at least a present simple tense for all test names.

File: eo-parser/src/test/java/org/eolang/parser/PacksTest.java (hunk 2 above)
Comment (yegor256) [reply eo-parser/src/test/java/org/eolang/parser/PacksTest.java:40]: @volodya-lombrozo I didn't touch the names of methods, just modified their implementation, that's why... I will modify them now

**Comment by yegor256** [comment]
@volodya-lombrozo I renamed the methods as you suggested

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**